near-contract-standards/src/non_fungible_token/token.rs

```diff
@@ -0,0 +1,15 @@
+use crate::non_fungible_token::metadata::TokenMetadata;
+use near_sdk::serde::{Deserialize, Serialize};
+use near_sdk::AccountId;
+use std::collections::HashMap;
+
+pub type TokenId = String;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[serde(crate = "near_sdk::serde")]
+pub struct Token {
+    pub token_id: TokenId,
+    pub owner_id: AccountId,
+    pub metadata: Option<TokenMetadata>,
+    pub approved_account_ids: Option<HashMap<AccountId, u64>>,
+}
```

Comment:
I think docs here would be really useful, if the struct is imported, it might not even be clear to a developer what type of token this is